app/models/work.rb

```diff
@@ -0,0 +1,21 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  validates :title, presence: true
+
+  def self.sort_works(category)
+    works = Work.where(category: category)
+    sorted_works = works.sort_by {|work| work.votes.count}
+    return sorted_works.reverse
+  end
+
+  def self.list_top_ten(category)
+    works = Work.sort_works(category)
+    return works[0..9]
+  end
```

Comment:
I love the way these model methods build on each other, each adding a small bit of functionality. This is a great example of functional decomposition.